libraries/adaptive-expressions/tests/expressionParser.test.js

```diff
@@ -37,10 +37,10 @@ const dataSource = [
     ['`hi ${string(\'jack`\')}`', 'hi jack`'],
     ['`\\${world}`', '${world}'],
     ['length(`hello ${world}`)', 'hello world'.length],
-    ['json(`{"foo":"${hello}","item":"${world}"}`).foo', 'hello'],
```

Comment:
>foo [](start = 14, length = 3)

This is offensive?  Everything I could find called it a metasyncratic place holder which hardly sounds offensive.